components/contactus/contact.js

```diff
@@ -3,21 +3,24 @@ import FirstStep from './firstStep'
 import Button from './button'
 import SecondStep from './secondStep'
 import ThirdStep from './thirdStep'
+import { useForm } from 'react-hook-form';
 
 const Form = () => {
     const [step,setStep] = React.useState(1);
+    const [formState , setFormState] = React.useState(1)
```

Comment:
import useState one time and use it directly, instead of using React.useState() each time.